docs/OTSubscriber.md

```diff
@@ -90,3 +91,33 @@ class App extends Component {
   }
 }
 ```
+
+## Manual rendering of streams
+
+It's possible to manually render streams, i.e. to allow touch interaction or provide individual styling for each `OTSubscriberView`.
+
+By using children as a render prop function, we can take care of rendering each `OTSubscriberView` individually, i.e.:
+
+```js
+<OTSubscriber
+  streamProperties={streamProperties}
+  >
+    {this.renderSubscribers}
+</OTSubscriber>
+```
+
+The render prop function could be written as follows:
```

Comment:
Could you mention that "to display the streams, pass the individual stream IDs into the `streamId` prop of the `OTSubscriberView` component, e.g.:" ?